src/main/java/com/opentable/db/postgres/embedded/PreparedDbProvider.java

```diff
@@ -33,69 +31,80 @@
 import com.google.common.base.Preconditions;
 import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Maps;
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
 
 import org.apache.commons.lang3.RandomStringUtils;
+import org.postgresql.ds.PGSimpleDataSource;
 
-public class EmbeddedPostgreSQLController
+public class PreparedDbProvider
 {
     private static final String JDBC_FORMAT = "jdbc:postgresql://localhost:%d/%s";
 
     /**
      * Each database cluster's <code>template1</code> database has a unique set of schema
      * loaded so that the databases may be cloned.
      */
-    @GuardedBy("EmbeddedPostgreSQLController.class")
-    private static final Map<Entry<URI, Set<String>>, Cluster> CLUSTERS = Maps.newHashMap();
+    @GuardedBy("PreparedDbProvider.class")
+    private static final Map<DatabasePreparer, PrepPipeline> CLUSTERS = new HashMap<>();
 
-    private final DatabaseMigrator migrator;
-    private final Cluster cluster;
+    private final PrepPipeline dbPreparer;
 
-    public EmbeddedPostgreSQLController(URI baseUrl, String[] personalities)
+    public PreparedDbProvider(DatabasePreparer preparer)
     {
         try {
-            cluster = getCluster(baseUrl, personalities);
+            dbPreparer = createOrFindPreparer(preparer);
         } catch (final IOException | SQLException e) {
             throw Throwables.propagate(e);
         }
     }
 
     /**
      * Each schema set has its own database cluster.  The template1 database has the schema preloaded so that
-     * each test case need only create a new database and not re-invoke Migratory.
```

Comment:
After removing it, did you grep for all strings "migratory"?  Just to make sure that you haven't missed anything.